.travis.yml

```diff
@@ -34,38 +43,23 @@ matrix:
       compiler: gcc-4.9
   fast_finish: true
 before_install:
-  # Pins the version of the java package installed on the Travis VMs
-  # and avoids a lengthy upgrade process for them.
-  - if [ $TRAVIS_OS_NAME = linux ]; then
-      sudo apt-mark hold oracle-java7-installer oracle-java8-installer;
-      sudo apt-get update;
-    elif [ $TRAVIS_OS_NAME = osx ]; then
+  - if [ $TRAVIS_OS_NAME = osx ]; then
       brew update;
     fi
+  - pip install --user -q --upgrade pip
+  # Add location of the pip packages to the path.
+  - PATH=$HOME/.local/bin:$PATH
 install:
-  - if [ $TRAVIS_OS_NAME = linux ]; then
-      sudo apt-get install xclip gdb;
-    elif [ $TRAVIS_OS_NAME = osx ]; then
+  - if [ $TRAVIS_OS_NAME = osx ]; then
       brew install gettext;
     fi
-before_script:
-  # Adds user to a dummy group.
-  # That allows to test changing the group of the file by `os_fchown`.
```

Comment:
Will this test still be correctly performed, or do we need a new solution for this?